console/src/dataSources/services/mysql/sqlUtils.ts

```diff
@@ -199,6 +199,15 @@ export const getAlterColumnCommentSql: DataSourcesAPI['getAlterColumnCommentSql'
     )} modify column \`${columnName}\` ${columnType} comment ${commentStr};`;
   };
 
+export const getAlterViewCommentSql: DataSourcesAPI['getAlterViewCommentSql'] =
+  ({ viewName, schemaName, comment }) => {
+    const commentStr = sqlEscapeText(comment);
+    return `alter view ${getMySQLNameString(
+      schemaName,
+      viewName
+    )} comment = ${commentStr};`;
+  };
```

Comment:
We can remove this. We don't have support for `mysql_run_sql` query type at the moment.

Reply:
@vijayprasanna13 Updated the PR to reflect the requested changes.